docs_user/assemblies/assembly_prepare_director_operator_and_rhoso_for_adoption_process.adoc

```diff
@@ -0,0 +1,190 @@
+ifdef::context[:parent-context: {context}]
+
+[id="prepare_director_operator_and_rhoso_for_adoption_process"]
+
+:context: migrating-databases
```

Comment:
```suggestion
:context: prepare-director-operator
```
The director operator assembly needs its own unique context.

Reply:
ok